app.py

```diff
@@ -58,11 +126,27 @@
     use_container_width=True,
 )
 
-st.header("Collaborative training participants")
 serialized_data, profiles = get_new_bubble_data()
+df_leaderboard = get_leaderboard(serialized_data)
 observable(
-    "Participants",
+    "_",
     notebook="d/9ae236a507f54046",  # "@huggingface/participants-bubbles-chart",
     targets=["c_noaws"],
-    redefine={"serializedData": serialized_data, "profileSimple": profiles},
+    redefine={"serializedData": serialized_data, "profileSimple": profiles, "width": 0},
 )
```

Comment:
Ideally, I would not have wanted to add this animation here. Unfortunately, if I remove it the animation with the bubbles in the upper right corner returns an error.

The consequence of this addition is an empty block at the bottom of the dashboard

![image](https://user-images.githubusercontent.com/55560583/144901510-2ef43435-9a93-427b-8cbf-9bfb75ea7e60.png)